src/main/java/org/cactoos/scalar/FirstOf.java

```diff
@@ -53,6 +53,19 @@
      */
     private final Scalar<T> fallback;
 
+    /**
+     * Constructor with default condition (always `true`).
+     * @param src Source iterable
+     * @param fbck Fallback used if no value matches
+     */
+    public FirstOf(final Iterable<T> src, final Scalar<T> fbck) {
```

Comment:
@iakunin let's also add a constructor that takes a `T` to ease use, WDYT?

Reply:
@victornoel do you mean `T` as a fallback, right?

Reply:
@iakunin yes exactly, sorry for the confusion :)

Reply:
@victornoel done =)